### PR TITLE
Process dataset elements with access children

### PR DESCRIPTION
**Pull request by lesserwhirls**
Fix Unidata/siphon#114
Fix Unidata/siphon#115

### CODE REVIEW

**Comment by lesserwhirls** [comment]
I'm guessing this will have some pep issues...

**Comment by dopplershift** [comment]
Looks like my quote-fixing spree is in conflict. Sorry, dude.

**Comment by lesserwhirls** [comment]
Nah, that was an easy fix. I'll up updating the PR in the next few minutes.

**Comment by lesserwhirls** [comment]
@dopplershift have you thought about turning off merge and squash+merge commits and just allowing the rebase and merge option?

**Comment by dopplershift** [comment]
I've fallen out of favor with the rebase and merge button--I like seeing the split and merge for each PR.

What I still want is the ability to rebase the branch from the web UI rather than the merge.

**Comment by dopplershift** [comment]
More pep8. 😢

File: siphon/tests/test_catalog.py [hunk 1]
```diff
@@ -135,3 +135,11 @@ def test_non_standard_context_path():
...
                 'P1A/A20020101.P1A.ANN_MIM_RMP.nc')
     assert ds.access_urls['OPENDAP'] == expected
+
+
+@recorder.use_cassette('cat_access_elements')
+def test_access_elements():
+    'Test parsing access elements in TDS client catalog'
+    url = 'http://oceandata.sci.gsfc.nasa.gov/opendap/SeaWiFS/L3SMI/2001/001/catalog.xml'
+    cat = TDSCatalog(url)
+    assert len(list(cat.datasets)) != 0
```
Comment (dopplershift) [inline siphon/tests/test_catalog.py:145]: *sigh* If we had only known and had written the test to check the correct length, which is 174.

**Pull request merged**